debian/rules

```diff
@@ -2,12 +2,14 @@
 # -*- makefile -*-
 
 export DH_OPTIONS
+PYTHON_BINARY := $(shell { command -v python3.4 || command -v python3.3 || command -v python2.7; } 2>/dev/null)
```

Comment:
@asottile if you are concerned about using different python versions, I can re-arrange them to use python2.7 by default.

However, I do believe there must be a way to build the package on a system that doesn't have python2.7 installed.